pyproject.toml

```diff
@@ -22,6 +22,7 @@ src = ["src/**/*.pyx"]
 #ANSIBLE_PYLIBSSH_TRACING = "1"
 #CFLAGS = "-DCYTHON_TRACE=1 ${CFLAGS}"
 LDFLAGS = "-lssh ${LDFLAGS}"
+CFLAGS="-I /usr/local/include ${CFLAGS}"
```

Comment:
I'm wondering this should be after the existing flags and not before :thinking:
Thoughts @Qalthos?

Reply:
I'm sure, you know better, where to include this setting.
Maybe there is a platform depended possibility?

Reply:
I've tried to find a more or less simple way to define the compile environment for a single platform (i.e. freebsd), but this quickly becomes complex:

- `CFLAGS` are overwritten in `tox.ini`, so a pass-through from the calling environment would require changes there.
- Another possibility is to define an `envlist` for the `tox.ini` target, which would multiply the size of `tox.ini`. 
- Providing platform based `extra_compile_args` in `[build_env]` sections of the `pyproject.toml` simply does not work (due to the more specific cythonize definitions.

Hence I'd would keep the proposed solution. `/usr/local/include` is not that uncommon, so that other distributions would profit, while the current distributions are not harmed.

Reply:
FWIW this is designed to allow passing though env vars from the calling shell. So yes, changing the `passenv` in `tox.ini` is reasonable in this case. If your system has env vars set properly already, tox shouldn't isolate from that. Although, if your use-case is just building the dists, you probably shouldn't use tox and do so via `pip install`/`pip wheel` or `python -m build`.

Anyway, I'll let @Qalthos decide on this, I think. I don't have a strong preference.

Reply:
Thinking about this more, maybe we shouldn't get into the business of hardcoding values that aren't directly related to our packaging efforts... Not on the native Python packaging level — the ability to pass those through seems like a good option. @donnerhacke perhaps try moving this to `tox`? Just `passenv` or `setenv` seem better suited.

Reply:
@donnerhacke so do you think using `passenv` would be helpful?